src/app/views/search-route/search-route.component.ts

```diff
@@ -19,6 +22,7 @@ export class SearchRouteComponent implements OnInit, AfterViewInit {
   @ViewChild('dateInput') dateInput: ElementRef;
   @ViewChild('timeInput') timeInput: ElementRef;
   @ViewChild('passengerInput') passengerInput: ElementRef;
+  @ViewChild('stagedRouteModal') stagedRouteModal: any;
```

Comment:
cound we try to find out the type, if not it's ok too

Reply:
ElementRef seems to work, but TemplateRef<object> as well. Not sure whats the preferred way.